### PR TITLE
mptcp: add delays before time-wait switch

### PR DESCRIPTION
**Pull request by matttbe**
Our internal CI at Tessares reported an issue with some fastopen tests where we expect to see the very end of a connection.

The error only happens with a debug kernel config and on a slow/old machine. I didn't manage to reproduce this on my side even if it is easy to reproduce the problem on this specific CI machine.

Here is an example of error we can see:

    client-TCP_FASTOPEN_CONNECT.pkt:22: error handling packet: live packet field ipv4_total_length: expected: 52 (0x34) vs actual: 60 (0x3c)
    script packet:  0.024743 . 2:2(0) ack 2 <nop,nop,TS val 101 ecr 700>
    actual packet:  0.024933 . 2:2(0) ack 2 win 256 <nop,nop,TS val 124 ecr 700,dss dack4 3007449510 flags: A>

In other words, the last ACK after the FINs exchange has MPTCP options.

The fix is similar to commit 2aa2bda ("mptcp: update fastclose tests") where Paolo modified `syscalls/connect_close_full.pkt` test to add some delays before injecting the last FIN+ACK. This makes sure the socket has switched to TIME_WAIT state where no MPTCP options are added.

### CODE REVIEW

**Review by dcaratti — APPROVED** [review]
LGTM. thanks!
